inventory/by_service/apache

```diff
@@ -0,0 +1,16 @@
+[apache_servers]
```

Comment:
Could this file use the group of groups pattern as well?

Reply:
Addressed in 8d9c4fcb37806a796bcabcfa895ff8bd752bf8bd. Thanks!